tests/test_insert.py

```diff
@@ -1,11 +1,20 @@
 from datetime import date
+from unittest import skipIf
 
 from tests.testcase import BaseTestCase
 from clickhouse_driver import errors
 from clickhouse_driver.errors import ServerException
 from tests.util import require_server_version
 
 
+try:
```

Comment:
Numpy/pandas-releated tests should be placed in `tests/numpy/*` and inherited from `NumpyBaseTestCase` see `tests/numpy/test_generic.py` as example.

This will do all the `PANDAS_IMPORTED` magic with skipping tests.